userspace/libscap/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-include_directories("${PROJECT_SOURCE_DIR}/common")
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}/../../common")
```

Comment:
Why is this necessary?

Reply:
My reading of the cmake docs (https://cmake.org/cmake/help/latest/variable/PROJECT_SOURCE_DIR.html) says that PROJECT_SOURCE_DIR can be influenced by the last project() command. Since libs might be included in other projects, I would think this is more reliable.

Reply:
I don't want to nitpick but if it's not strictly needed for Windows support, I'd rather leave it out of this PR (especially if it ends up squashed with all the other Windows changes).

If it doesn't break anything, by all means make it its own PR